index.js

```diff
@@ -166,6 +166,10 @@ var JUnitReporter = function (baseReporterDecorator, config, logger, helper, for
     var suite = suites[browser.id]
     var result = browser.lastResult
     if (!suite || !result) {
+      // Make sure we report any error messages to stdout if we are not going
+      // to create any XML.  See issue #97 for details.
+      process.stdout.write(allMessages.join() + '\n');
+
```

Comment:
Sorry, the karma project uses some sort of pseudo-coffee script style: no semicolons.